doc/LANGUAGE.md

```diff
@@ -345,26 +345,36 @@ Instruction                 | Flags   | Description
 
 The `const` instruction is used to introduce a constant value into the IR. The first version constructs a constant integer value, the second a constant integer signal, and the third a constant time value.
 
-    %result = const iN <int>
-    %result = const iN$ <int>
     %result = const time <time>
+    %result = const iN <int>
+    %result = const nN <enum>
+    %result = const lN <logic>
 
+- `time` is a time literal such as `1s`, `1s 2d`, or `1s 2d 3e`, where the real component may carry an SI suffix such as `as`, `fs`, `ps`, `ns`, `us`, `ms`, `s`.
```

Comment:
Excellent idea to allow for `s` as well.

Reply:
Credit where it's due: [opt_1.llhd](https://github.com/fabianschuiki/llhd/blob/master/test/opt_1.llhd) :)

Reply:
:smile: